pom.xml

```diff
@@ -394,22 +394,6 @@
                 </pluginManagement>
             </build>
             <dependencies>
-                 <dependency>
-                     <groupId>jakarta.xml.ws</groupId>
-                     <artifactId>jakarta.xml.ws-api</artifactId>
-                     <version>2.3.3</version>
-                 </dependency>
-                 <dependency>
-                     <groupId>jakarta.xml.bind</groupId>
-                     <artifactId>jakarta.xml.bind-api</artifactId>
-                     <version>2.3.3</version>
-                 </dependency>
-                 <dependency>
-                     <groupId>org.glassfish.jaxb</groupId>
-                     <artifactId>jaxb-runtime</artifactId>
-                     <version>2.3.2</version>
-                     <scope>test</scope>
-                 </dependency>
                  <dependency>
                      <groupId>com.sun.xml.messaging.saaj</groupId>
```

Comment:
This needs to be updated to 2.0.1 and some compile time dependencies fixed:

[ERROR] /Users/coheigea/src/apache/ws-wss4j/ws-security-common/src/test/java/org/apache/wss4j/common/util/SOAPUtil.java:[24,22] package javax.xml.soap does not exist
[ERROR] /Users/coheigea/src/apache/ws-wss4j/ws-security-common/src/test/java/org/apache/wss4j/common/util/SOAPUtil.java:[45,20] cannot find symbol
  symbol:   class MessageFactory
  location: class org.apache.wss4j.common.util.SOAPUtil

Reply:
Done